lib/config/options/index.ts

```diff
@@ -1242,7 +1242,7 @@ const options: RenovateOptions[] = [
     default: {
       branchTopic: '{{{depNameSanitized}}}-digest',
       commitMessageExtra: 'to {{newDigestShort}}',
-      commitMessageTopic: '{{{depName}}} commit hash',
+      commitMessageTopic: '{{{depName}}} commit digest',
```

Comment:
Doesn't seem to give a good flow. What is "commit digest"?
I think the words "commit hash" and "docker (image) digest" exist, but not "commit digest".